templates/base.html

```diff
@@ -20,7 +20,7 @@
         <div style="text-align:center;">
             &copy; 2023
             <a href="/" class="home"><img id="home" src="{{ url_for('static', path='/home.png') }}"></a>
-            Version: 1.4.5
+            Version: 1.4.6
```

Comment:
update this shit so the label doesn't say `frontend` if i only changed a `base.html` file from bumping the repo